### DISCUSSION TITLE
How to use tailwind pagintation?

### DISCUSSION THREAD

**Discussion by lucidpolygon** [statamic/cms#7311] [General]
@php($tag = Statamic::tag('collection:blogs')->param('title:contains', 'Laravel')->paginate(2)->as('entries')->fetch())
            @foreach($tag['entries'] as $entry)
            @include('blog.partials.article')
            @endforeach

        {{ $tag['paginate']['auto_links'] }}
        
        This works pretty well on blade. The only thing is that the pagination is plain HTML and not a tailwind. How to use tailwind pagination?
        
        Thanks very much.

**Comment by jackmcdade** [comment]
Here's how you'd do a custom pagination with your HTML however you want: https://statamic.dev/tags/collection#pagination-examples

**Reply by lucidpolygon** [reply]
Had seen this. So that's the only way. Cool. Thanks.

**Reply by lucidpolygon** [reply]
Can I add one more quick question.
Can I make this query in the controller? I can't figure out a way to query using tags.

            @php($tag = Statamic::tag('collection:blogs')->param('title:contains', 'shopify')->paginate(2)->as('entries')->fetch())


something like this:

Entry::query()
            ->where('collection', 'blogs')
            ->whereIn('tags', ['Laravel '])
            ->get()

**Comment by jackmcdade** [comment]
Docs on that are right here: https://statamic.dev/content-queries#like-amp-not-like

Should be able to do something like:

```
$entries = Entry::query()
->where('collection', 'blogs')
->where(title', 'like', '%Laravel%')
->paginate(2)
->get()
```

**Reply by lucidpolygon** [reply]
This is for the title right. But what I want is query via taxonomy.

Blogs have taxonomy (tags).

I am querying all blogs that are tagged with Laravel. (Title of the blog may not mention Laravel)

**Reply by jackmcdade** [reply]
Ah that's different, why didn't you ask that before?

```
Entry::query()
->where('collection', 'blogs')
->whereTaxonomy('tags::laravel')
->get()
```

**Reply by lucidpolygon** [reply]
Just got it done. Thanks, @jackmcdade and whoever it is replied to me in Twitter :) You guys are awesome.

This is how I got it working for the blade as well

```
 @foreach(Statamic::tag('taxonomy:tags')->param('title:contains','Laravel')->as('entries')->fetch()['entries'] as $entry)
    @foreach($entry->entries as $entry)
        @include('blog.partials.article')
    @endforeach
@endforeach
```
